sched/semaphore/sem_wait.c

```diff
@@ -126,6 +171,13 @@ int nxsem_wait_slow(FAR sem_t *sem)
 
       rtcb->waitobj = sem;
 
+      /* In case of a mutex, store the previous holder in the task's list */
+
+      if (mutex)
```

Comment:
let's remove ALL holder related function call in the mutex case

Reply:
what?